web-animations.js

```
@@ -248,6 +250,7 @@ Player.prototype = {
     } finally {
       exitModifyCurrentAnimationState(this._hasTicked);
     }
+    this._checkForHandlers();
```

Done.

Seeing this here, I'm a little concerned that try { } finally without catch reporting is going to make modifying the code harder as exceptions are silently swallowed. Do you think we could add some catches?